tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/metadata.py

```diff
@@ -27,24 +26,32 @@ def test(ctx: click.Context):
 
 @test.command(help="Run unit tests for the metadata service library.")
 @click.pass_context
+@pipeline_command
 def lib(ctx: click.Context):
-    logger.info("Running metadata service lib unit tests...")
-    try:
-        pipeline_success = anyio.run(
-            run_metadata_lib_test_pipeline,
-            ctx.obj["is_local"],
-            ctx.obj["git_branch"],
-            ctx.obj["git_revision"],
-            ctx.obj.get("gha_workflow_run_url"),
-            ctx.obj.get("pipeline_start_timestamp"),
-            ctx.obj.get("ci_context"),
-        )
-        if not pipeline_success:
-            raise dagger.DaggerError("Metadata Service Lib Unit Test Pipeline failed.")
-
-    except dagger.DaggerError as e:
-        click.secho(str(e), err=True, fg="red")
-        return sys.exit(1)
+    return anyio.run(
+        run_metadata_lib_test_pipeline,
+        ctx.obj["is_local"],
+        ctx.obj["git_branch"],
+        ctx.obj["git_revision"],
+        ctx.obj.get("gha_workflow_run_url"),
+        ctx.obj.get("pipeline_start_timestamp"),
+        ctx.obj.get("ci_context"),
+    )
+
+
+@test.command(help="Run unit tests for the metadata service orchestrator.")
```

Comment:
nit: I think it does not matter to detail which  nature of test you run. If you add integrations tests I bet you'll make unit tests and integration tests run in the same command.
```suggestion
@test.command(help="Run tests for the metadata service orchestrator.")
```